apps/frontend/app/AdminChoachRequest/hooks/useCoachRequestUpdate.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react";
+import { TrainerRequest } from "../../types";
+
+export const useCoachRequestUpdate = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+
+  const updateTrainerRequest = async (id: string, updatedData: Partial<TrainerRequest>) => {
+    setIsLoading(true);
+    setError(null);
+
+    try {
+      const response = await fetch(`http://localhost:4444/trainer-requests/${id}`, {
```

Comment:
in utils there is a file for connections and requests to the back end, place it there.